catalyst-gateway/bin/src/db/index/schema/cql/txo_assets_by_stake_table.cql

```diff
@@ -6,12 +6,12 @@ CREATE TABLE IF NOT EXISTS txo_assets_by_stake (
     slot_no         varint,   -- slot number the txo was created in.
     txn             smallint, -- Which Transaction in the Slot is the TXO.
     txo             smallint, -- offset in the txo list of the transaction the txo is in.
-    policy_id       blob,     -- asset policy hash (id) (28 byte binary hash)
-    policy_name     text,     -- name of the policy (UTF8) TODO: https://github.com/input-output-hk/catalyst-voices/issues/1121
+    asset_id        blob,     -- asset policy hash (id) (28 byte binary hash)
```

Comment:
```suggestion
    policy_id        blob,     -- asset policy hash (id) (28 byte binary hash)
```

This was correctly called "Policy ID" it is the hash of the Asset policy script.  So Policy ID is correct.